rdrf/rdrf/templates/registration/login_assistance_form.html

```diff
@@ -0,0 +1,20 @@
+{% extends "rdrf_cdes/base.html" %}
+{% load i18n %}
+
+{% block content %}
+
+<br>
+
+<div class="alert alert-warning">
+  {% trans "Trouble signing in? Did you forgot your password or your account has been locked for security reasons? Enter your email address below, and we'll email you instructions on what to do." %}
```

Comment:
I don't think we need to mention account lock here.

"Trouble signing in? Did you forgot your username or password? Enter your email address below, and we'll email you instructions on what to do."